### DISCUSSION TITLE
KM271 + R4111

### DISCUSSION THREAD

**Discussion by Robin19777** [dewenni/ESP_Buderus_KM271#105] [Q&A]
Hallo zusammen,

Bei meiner Heizungsanlage ist ein Logamatic r4111 verbaut. Bislang habe ich nichts gefunden, womit ich die Anlage Smart steuern und optimieren könnte. Ich weiß nicht wie die Logamatic r4111 aufgebaut ist. Kann man mit diesem KM 271 Board dies ermöglichen? Sollte ich dann die "Option 2 - ESP32 with original Buderus KM271" wählen und über die serielle Schnittstelle gehen?

Ich hoffe Ihr könnt mir helfen oder wisst eine Alternative.

Grüße, 
Robin

**Comment by the78mole** [comment]
Hallo Robin,

wenn die Original KM271 reinpasst, dann passt auch meine KM271-WiFi. Den r4111 kenne ich überhaupt nicht. Da die Informationen von Buderus zu kompatiblen Erweiterung nichtmal spärlich sondern anscheinen nicht vorhanden sind, ist es am einfachsten, Du öffnest mal das Gehäuse. Es sind nur 2 Schrauben (Oberseite), die man lösen muss. Wenn da ein Slot ist (wie man auf dem Bild in meinem Blog erkennen kann), dann sollte es funktionieren.

Grüße,
Daniel

**Comment by thjm** [comment]
Hi Robin,

in Kapitel 5 der Service-Anleitung ist kein KM271 aufgeführt, ich sehe da schwarz, aber reinschauen und sich vergewissern, ist auch eine Option.

Siehe: https://www.intec-heizung.de/media/pdf/93/4f/bc/Buderus-Logamatic-4111-4112-4116-Serviceanleitung.pdf

Gruß
Hermann-Josef

**Comment by the78mole** [comment]
Hm, das sieht tatsächlich dünn aus... Die einzige sinnvolle (externe) Schnittstelle, die ich erkenne, ist das EIB-Modul, aber da bin ich mir ziemlich sicher, dass das wenig/keine Ähnlichkeit zum KM271 hat.
